packages/build/src/plugins_core/deploy/build_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+option go_package = "github.com/netlify/buildbot/buildservice/proto";
+
+package buildservice;
+
+// Service to communicate between Netlify Build and the buildbot
+service BuildService {
+  // Request the buildbot to deploy the site
+  rpc RunPostDeploy(PostDeployRequest) returns (PostDeployResponse) {}
```

Comment:
Taken from https://github.com/netlify/buildbot/pull/1122

`RunBuild()` not implemented yet, so we can split this feature into several PRs.

I added `await_live` to the request. This is needed for the moment. When Netlify Build becomes the server instead of the client, this won't be needed anymore. This is boolean instead of an enum, since it might be simpler.

The response includes a `succeeded` boolean, to make it simpler to know whether `error` and `error_type` should be checked. `error_type` is a string to re-use the existing logic and avoid mapping enums to strings.